notebooks/data-sources/TestGrid/metrics/probability_to_fail.ipynb

```diff
@@ -0,0 +1,1091 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    AUTOMATION = os.getenv("True")
</pre><p>Shouldn't this be <code>AUTOMATION = os.getenv("IN_AUTOMATION")</code>  <span class="ql-cursor">﻿</span></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/310/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
:+1: